### ISSUE TITLE
import group doesn't set scope

### ISSUE THREAD

**Issue by drdamour** [microsoft/terraform-provider-azuredevops#345]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* `azuredevops_group`

### Terraform Configuration Files

```hcl
resource "azuredevops_group" "crush" {
  scope        = azuredevops_project.opsys.id
  display_name = "Crush"
  description  = <<-EOF


    Managed by TF
EOF

}
```

### Expected Behavior

after importing an immediate apply should not recreate the resource do to scope being altered

### Actual Behavior

tf plan shows that a scope is causing a forced replacement

### Steps to Reproduce

import a project scoped group then tf plan it.

1. `terraform apply`

### Important Factoids

Removing the scope stanza from TF makes the replace go away...so i think when refreshing state it's not checking the scope either as omitting the scope stanza probably should show a problem

modifying the state file with the scope makes the replacement go away too as expected.

**Comment by xuzhang3** [comment]
Hi @drdamour Thanks for your feedback. The `scope` is the query parameter, should not be used to check the resource difference,  I have submit a PR #347  to fix this issue.

**Comment by xuzhang3** [comment]
Close this issue, feel free to open another issue if you have questions.

**Comment by drdamour** [comment]
@xuzhang3 i'm not sure this was fixed,  using 0.1.5 i'm now getting

![image](https://user-images.githubusercontent.com/1514496/121739026-49730500-cab8-11eb-923c-a06ebcb36340.png)

**Comment by gdud** [comment]
I have the same issue as @drdamour in version 0.1.5 ... 

Scope is set to null in a state file.

**Comment by xuzhang3** [comment]
@gdud  Are you trying create a new group or import a group? can you share your configuration?

**Comment by drdamour** [comment]
@xuzhang3 i just had a need to do this again..and it's still and issue using 0.1.7

i import he group,then when i go to apply i get force replacement because of scope.  in the state file it's empty

**Comment by drdamour** [comment]
can we reopen this plz

**Comment by Annesars90** [comment]
What is the status on this issue? I'm also getting the same problem. This is my process to test:
- Deploy a new group with terraform (incl scope).
- Remove group from tf state so it only exists is DevOps
- Import group to tf state, which leads to scope = null in the statefile
- Trying to redeploy group forces replacement because of scope

**Comment by xuzhang3** [comment]
@Annesars90  fixed in #519 . This `scope` is used to get the `scope descriptor` and the `scope descriptor` will not return by service,  this property should not be compared during the plan/apply

**Comment by drdamour** [comment]
confirmed this is now working in 0.2.0, thanks!
